orchagent/pfc_restore_cisco-8000.lua

```diff
@@ -44,7 +44,7 @@ for i = n, 1, -1 do
         and (debug_storm ~= "enabled")
         -- DEBUG CODE END.
         then
-            if time_left <= 0 then
+            if time_left <= poll_time then
```

Comment:
We had this discussion during #1748 review. https://github.com/Azure/sonic-swss/pull/1748#discussion_r695934838
What has changed now?

Reply:
@alpeshspatel , I am not clear why this is changed to compare with poll_time. in 1748, the change was made to compare with 0 and you had provided a small writeup for the reason.

Reply:
resolved offline